contributors.txt

```diff
@@ -92,4 +92,5 @@ YYYY/MM/DD, github id, Full name, email
 2019/08/11, KarstenRa, Karsten Rathlev, krathlev(at)gmail.com
 2019/08/15, phdussud, Patrick Dussud, phdussud[at]hotmail.com
 2019/08/18, manuc66, Emmanuel Counasse, manuc66(at)gmail.com
+2019/08/25, kvanttt, Ivan Kochurkin, kvanttt(at)gmail.com
```

Comment:
Could you create a dedicated PR for signing in order to avoid merge conflicts by merging it quickly independent of this PR.

Reply:
No problem.

Reply:
Please send a PR with your sign off first before we merge this.